demosplan/DemosPlanCoreBundle/Controller/User/DemosPlanOrganisationAPIController.php

```diff
@@ -255,8 +255,17 @@ public function wipeOrgaAction(UserHandler $userHandler, string $id): APIRespons
     {
         $orgaId = $id;
         try {
-            $isOrgaDeleted = $userHandler->wipeOrganisationData($orgaId);
-            if ($isOrgaDeleted) {
+            $result = $userHandler->wipeOrganisationData($orgaId);
+
+            if (is_array($result)) {
+                // Handle errors here
+                foreach ($result as $error) {
+                    $this->messageBag->add('error', $error);
+                }
+
+                return $this->renderEmpty(Response::HTTP_UNAUTHORIZED);
+            } else {
```

Comment:
I think we don't need the else here for confirmation

Reply:
done in [5c3c6df](https://github.com/demos-europe/demosplan-core/pull/2681/commits/5c3c6df381e845a5949f2d7998d74338942e678a)